pkg/pillar/pubsub/pubsub_test.go

```diff
@@ -6,24 +6,58 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const (
-	agentName = "agentName"
-)
-
 type Item struct {
 	aString string
 }
 
-var item = Item{
-	aString: "aString",
+var (
+	item = Item{
+		aString: "aString",
+	}
+)
+
+type EmptyDriver struct{}
```

Comment:
In the Empty driver, where are we storing the Data? In the Same keyMap as in Publisher / Subscriber??

I would rename it as TestDriver. We may add more information needed for tests there..

Reply:
Yes. This is actually why I created it. It shows that you can still publish, but the data ends up in the in-memory keymap, and doesn't actually get published anywhere. 

> I would rename it as TestDriver. We may add more information needed for tests there..
 
Sure. I will try to get it done here, but if not, another PR would be easy. It is used entirely in this file.